Sources/Sentry/SentryUIViewControllerPerformanceTracker.m

```diff
@@ -15,6 +15,9 @@
 static NSString *const SENTRY_UI_PERFORMANCE_TRACKER_LAYOUTSUBVIEW_SPAN_ID
     = @"SENTRY_UI_PERFORMANCE_TRACKER_LAYOUTSUBVIEW_SPAN_ID";
 
+static NSString *const SENTRY_UI_PERFORMANCE_TRACKER_VIEWAPPEARING_SPAN_ID
+    = @"SENTRY_UI_PERFORMANCE_TRACKER_LAYOUTSUBVIEW_SPAN_ID";
```

Comment:
`h`: I think this should be. I wonder why the tests are not failing. Can we write a test that fails if this is messed up?
```suggestion
    = @"SENTRY_UI_PERFORMANCE_TRACKER_VIEWAPPEARING_SPAN_ID";
```

Reply:
Good point.